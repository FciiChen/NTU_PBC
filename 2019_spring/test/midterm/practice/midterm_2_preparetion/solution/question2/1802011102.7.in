IZS, oNm,8qOJ7Oil,YKBPYvHaFE,yhHHdq
SLbL8LTiLv, TRZwU,0kb8AE,KU6n1ck6HD,t2qv
CebmzI, j3CPinHwXA,Bo4spY,fBvU,zEO
B0auuNukD, ZAZCs2,xdpC,06lZ0,kzPGowNrgD
UFgDCAI, 42Ota88NQL,ciwbvSGG7t,txg0,jeU7BPfBT
DwuD5jSQmS, aeDTS,xgEev,hMPVHp6,7WeD0qud
BREAK
B0auuNukD ,  lvu
CebmzI ,  BjK
SLbL8LTiLv ,  IxQysGri
DwuD5jSQmS ,  070FdwU
UFgDCAI ,  Dg8cbjhr
IZS ,  wiaL
BREAK
