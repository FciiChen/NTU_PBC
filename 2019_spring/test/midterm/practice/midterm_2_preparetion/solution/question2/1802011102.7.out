IZS        oNm        8qOJ7Oil   YKBPYvHaFE yhHHdq     wiaL      
SLbL8LTiLv TRZwU      0kb8AE     KU6n1ck6HD t2qv       IxQysGri  
CebmzI     j3CPinHwXA Bo4spY     fBvU       zEO        BjK       
B0auuNukD  ZAZCs2     xdpC       06lZ0      kzPGowNrgD lvu       
UFgDCAI    42Ota88NQL ciwbvSGG7t txg0       jeU7BPfBT  Dg8cbjhr  
DwuD5jSQmS aeDTS      xgEev      hMPVHp6    7WeD0qud   070FdwU   
